### PR TITLE
Support s3a hadoop file system.

### PR DESCRIPTION
**Pull request by kecaps**
Added tests to run through both with s3n and s3a
Updated hadoop-aws and jets3t packages.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -99,7 +99,7 @@
         <dependency>
             <groupId>net.java.dev.jets3t</groupId>
             <artifactId>jets3t</artifactId>
-            <version>0.7.1</version>
+            <version>0.9.3</version>
```
Comment (pgarbacki) [inline pom.xml]: Other people identified issues with upgrading jets3t lib. You may want to follow https://groups.google.com/forum/#!topic/secor-users/6TlNaKuPLwA

**Comment by pgarbacki** [comment]
@kecaps this PR broke the travis build. I'll have to revert it.

**Comment by pgarbacki** [comment]
https://travis-ci.org/pinterest/secor/builds/78074163

**Comment by kecaps** [comment]
The Travis ci testing has a timeout of 50 minutes. Since I doubled the
testing for both file systems it is exceeding the timeout. Would you prefer
removing test coverage or increasing the Travis timeout?

It may be fairly redundant to do all the tests for both file systems since
I think the s3 interaction of putting a file is the same in a lot of the
test scenarios.

**Comment by pgarbacki** [comment]
@kecaps let's increase the timeout for now. We can remove the tests later when the runtime becomes unbearable.

**Comment by kecaps** [comment]
I took a quick look at travis-ci documentation. It doesn't look like there
is a way to configure travis-ci to allow a build longer than 50 minutes:

http://docs.travis-ci.com/user/customizing-the-build/#Build-Timeouts
https://github.com/travis-ci/travis-ci/issues/2736

**Comment by pgarbacki** [comment]
@kecaps let's remove the tests then

**Pull request merged**